tests/Makefile

```diff
@@ -0,0 +1,72 @@
+##========================= Terraform Tests =========================#
+include ./config.mk
+
+tf-tests-up:
+	@cd ../ && docker build . -q -f ./tests/terraform/scripts/Dockerfile.build -t rke2-tf
```

Comment:
It might be nice to be able to choose the tag name so that you can, for example, have like `rke2-tf-rhel`, `rke2-tf-ubuntu`, etc. and run multiple configurations locally with different configs but not have to rebuild between runs. Open to discussion for what people would prefer with that!

Reply:
@rancher-max  great point. i am totally in favor of adding a   rke2-tf-${TAGNAME}

Reply:
@rancher-max  agreed!!

Reply:
done